pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -86,7 +86,9 @@ def pre_optimizer_step(
     def post_optimizer_step(self, optimizer: Optimizer, optimizer_idx: int) -> None:
         """Hook to do something after each optimizer step."""
 
-    def clip_gradients(self, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = float(2.0)) -> None:
+    def clip_gradients(
+        self, model: Any, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = float(2.0)
```

Comment:
can we drop these formatting changes, this is already a big PR :]

Reply:
I had to add model to the signature, this is more than just the formatting